.ci/docker/common/install_cudnn.sh

```diff
@@ -5,11 +5,11 @@ if [[ ${CUDNN_VERSION} == 8 ]]; then
     mkdir tmp_cudnn
     pushd tmp_cudnn
     if [[ ${CUDA_VERSION:0:4} == "12.1" ]]; then
-        CUDNN_NAME="cudnn-linux-x86_64-8.9.2.26_cuda12-archive"
+        CUDNN_NAME="cudnn-linux-x86_64-8.9.7.29_cuda12-archive"
```

Comment:
Please update only 11.8 cudnn version. At this point we want to upgrade only cuda 11.8 cudnn.

Reply:
@ptrblck Sorry yes, issue exist in cudnn 11, my bad https://pypi.org/project/nvidia-cudnn-cu12/#files. I do see  ``cudnn 8.9.7.29`` available.